### PR TITLE
add throttle env vars

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: ndoh_hub/settings.py [hunk 1]
```diff
@@ -211,8 +211,8 @@
     "DEFAULT_FILTER_BACKENDS": ("django_filters.rest_framework.DjangoFilterBackend",),
     "DEFAULT_THROTTLE_CLASSES": ["rest_framework.throttling.ScopedRateThrottle"],
     "DEFAULT_THROTTLE_RATES": {
-        "covid19triage.create": "30/second",
-        "covid19triage.list": "60/minute",
+        "covid19triage.create": os.environ.get("CREATE_THROTTLE_RATE", "30/second"),
```
Comment (erikh360) [inline ndoh_hub/settings.py]: Can we add `covid19triage` in this env var name please? if we want to start throttling more endpoints its going to get confusing

**Review by erikh360 — COMMENTED** [review]
Just the one comment for both

File: ndoh_hub/settings.py (hunk 1 above)
Comment (codiebeulaine) [reply ndoh_hub/settings.py]: yep

**Review by erikh360 — APPROVED** [review]
happy if you fix the lint errors

**Pull request merged**